src/WP_CLI/CommandWithUpgrade.php

```diff
@@ -471,17 +471,10 @@ protected function has_update( $slug ) {
 	/**
 	 * Get the available update info
 	 *
-	 * @param string $slug The plugin/theme slug
-	 *
-	 * @return array|null
+	 * @return array
```

Comment:
`get_site_transient()` will either return the value of the transient (which seems to be an object here, not an array, but generally `mixed`) or `false` if the transient was not found.
You should either adapt the docblock to be `mixed` or modify the value before returning it.